### PR TITLE
[WIP] MAGECLOUD-3378 scaled architecture

### PR DESCRIPTION
**Pull request by hguthrie**
DRAFT—The purpose of this PR is to present the current scaled architecture topic for technical review. There are many points that are not complete.

- [ ] Complete topic, or add further topics and updates
- [ ] Load balance incomplete
- [ ] Managing nodes incomplete
- [ ] Create 2.3 symlink
- [ ] Update TOC (remove 2.3 restriction)
- [ ] Technical review
- [ ] Link/lint/image checks

https://devdocs.magedevteam.com/902/guides/v2.2/cloud/architecture/architecture-scaling.html

### CODE REVIEW

File: guides/v2.2/cloud/architecture/architecture-scaling.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+---
+group: cloud-guide
+title: Scaling architecture
+functional_areas:
+  - Cloud
+---
+
+The Cloud infrastructure scales according to your resource needs to achieve greater efficiency. The {{site.data.var.ece}} auto-scaling monitors your applications and automatically adjusts capacity to maintain steady, predictable performance. Converting to this architecture helps to mitigate problems, such as latency, large spikes in traffic, or running out of storage.
```
Comment (YPyltiai) [inline guides/v2.2/cloud/architecture/architecture-scaling.md]: No `auto-scaling`. It is scripted and we monitor when we need to initiate it but it's not an auto-scaling.

File: guides/v2.2/cloud/architecture/architecture-scaling.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+### Database tier scaling
+
+There are 3 nodes for the database and services. When the database (db) tier approaches capacity, the only way to scale is to increase the server size, such as boosting the CPU power and memory. Capacity is limited to the size of the node that is available. Because the database cluster is designed for high availability, you cannot scale horizontally in a reliable way with the technologies used.
+
+![Database tier scaling]
+
+Further optimizing performance includes routing traffic. You can route traffic based on the node type. For example, you can serve PHP traffic on the database node or you can isolate the database node from the PHP traffic.
```
Comment (YPyltiai) [inline guides/v2.2/cloud/architecture/architecture-scaling.md]: Please add that by default DB nodes don't serve PHP traffic.

File: guides/v2.2/cloud/architecture/architecture-scaling.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+![Web tier scaling]
+
+This complements the vertical scaling provided by the database tier. (Introduce the concept of balancing...)
+
+Consider the following when scaling the web tier:
+
+-  You must use the same instance type and size for each node. Examples include:
+   -  all `M4 xlarge` for WEB
+   -  all `C4 xlarge` for DB
+-  Instances of a specific type and size must be available when scaling.
```
Comment (YPyltiai) [inline guides/v2.2/cloud/architecture/architecture-scaling.md]: Internal restriction, should be removed. We handle it without issues

File: guides/v2.2/cloud/architecture/architecture-scaling.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+-  Must balance the usage of instance types & sizes. (how to describe this?? )
```
Comment (YPyltiai) [inline guides/v2.2/cloud/architecture/architecture-scaling.md]: Remove too. Internal one too.

File: guides/v2.2/cloud/architecture/architecture-scaling.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+
+## Managing nodes
```
Comment (YPyltiai) [inline guides/v2.2/cloud/architecture/architecture-scaling.md]: Remove everything below this point. We'll add it back separately

**Comment by hguthrie** [comment]
I am going to close this for now. There is more work to do and I don't want to keep this PR up. Will create a new PR when a second draft is available for review.

**Pull request closed without merging**